modules/zip/src/parse-zip/cd-file-header.ts

```diff
@@ -188,3 +189,167 @@ const findExpectedData = (zip64data: Zip64Data): {length: number; name: string}[
 
   return zip64dataList;
 };
+
+/** info that can be placed into cd header */
+type CDGenOptions = {
+  /** CRC-32 of uncompressed data */
+  crc32: number;
+  /** File name */
+  fileName: string;
+  /** File size */
+  length: number;
+  /** Relative offset of local file header */
+  offset: number;
+};
+
+/**
+ * generates cd header for the file
+ * @param options info that can be placed into cd header
+ * @returns buffer with header
+ */
+export const generateCdHeader = (options: CDGenOptions): ArrayBuffer => {
```

Comment:
- Use `function` keywors
- I recommend preserving capitalization rather than decapitalizing to fit the camelCase.
- Also be consistent with function names and option types
```suggestion
export function generateCDHeader(options: GenerateCDOptions): ArrayBuffer {
```